.github/workflows/ci.yml

```diff
@@ -20,8 +20,8 @@ env:
   MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
   GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
   GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
-  RELEASE_VERSION: "7.4.0-A1"
-  DEVELOPMENT_VERSION: "7.4.0-A2-SNAPSHOT"
+  RELEASE_VERSION: "7.4.0"
+  DEVELOPMENT_VERSION: "7.4.0-SNAPSHOT"
```

Comment:
Since we're releasing `7.4.0` GA with this commit I believe it doesn't make sense to keep using `7.4.0-SNAPSHOT` as a development version. This should be bumped to either `7.5.0-SNAPSHOT` or `23.1.0-SNAPSHOT` depending on the next expected version of ACS.